### DISCUSSION TITLE
It's not clear if a repo is a fork

### DISCUSSION THREAD

**Discussion by WuerfelDev** [community/community#34] [Mobile] [answered]
When viewing a repo it doesn't show that it's a fork, and also not how much it is behind its origin (mobile webpage screenshot for comparison)
![Screenshot_20201026-111120](https://user-images.githubusercontent.com/12686819/97160782-22f89780-177d-11eb-9031-a4c521c1cd90.png)
![Screenshot_20201026-111708](https://user-images.githubusercontent.com/12686819/97160801-268c1e80-177d-11eb-8562-361828117858.png)

**Comment by brianlovin** [comment, marked as answer]
Thanks for the note here, totally agree that the profile view is confusing when we miss that metadata! I've opened an issue for our team to track this and hopefully get a fix out quickly 😊

**Reply by jsoref** [reply]
This appears to be resolved:

![Screenshot_20230111-091231.png](https://user-images.githubusercontent.com/2119212/211828136-e6285229-bbd3-4cb2-802a-a132b86c6d98.png)
